keyboards/alk0/cornedeon/keymaps/vial/rules.mk

```
@@ -0,0 +1,14 @@
+VIA_ENABLE          = yes
+VIAL_ENABLE         = yes
+LTO_ENABLE          = no
+
+RGBLIGHT_ENABLE     = no
+RGB_MATRIX_ENABLE   = yes # Can't have RGBLIGHT and RGB_MATRIX at the same time.
```

```suggestion
RGB_MATRIX_ENABLE   = yes
```
Setting `"lighting": "vialrgb"` in the `vial.json` without enabling VialRGB will cause a runtime error which makes the Vial GUI unusable, so if someone wants to switch to RGBLight they can't just switch the setting in `rules.mk` anyway.